src/languages/en/ui.js

```diff
@@ -32,7 +32,7 @@ export const LAST_NAME = 'Last Name';
 export const ROLE = 'Role';
 export const EMAIL = 'Email';
 export const JOB_TITLE = 'Job Title';
-export const WEEKLY_COMMITTED_HRS = 'Weekly Committed Hrs';
+export const WKLY_COMMITTED_HRS = 'Wkly Committed Hrs';
```

Comment:
Try to be consistent when naming variables, not a big issue as of right now. Your variables seemed to be mostly spelled right, but keep in mind to spell variables right.

Reply:
I named weekly committed hours based on the requirement from Jae. After abbreviation， it possible to  look better in one line.

Reply:
I guess you mean the WKLY_COMMITTED_HRS, I use abbreviation based on the requirement from Jae. So they all it on one line.